src/components/MediaEdit/MediaEditPresentation.js

```diff
@@ -41,93 +49,137 @@ function MediaEditPresentation({ dataToEdit, submitHandler, backHandler }) {
+          ) : (
+            <div className="m-6 col-span-6 max-h-1/3-screen rounded-lg overflow-hidden">
+              {mediaType === IMAGE && (
+                <img
+                  src={getMediaPath({
+                    mediaObject: dataToEdit,
+                    type: IMAGE,
+                  })}
+                  alt={dataToEdit?.title}
+                  className="object-contain w-full"
+                />
+              )}
+              {mediaType === VIDEO && (
+                <video
+                  className="w-full aspect-video"
+                  src={getMediaPath({
+                    mediaObject: dataToEdit,
+                    type: VIDEO,
+                  })}
+                  controls
+                />
+              )}
+            </div>
+          )}
 
-          <div className="col-span-12">
-            <Form.RadioButtons
-              label="Include in games?"
-              control={control}
-              errors={errors}
-              nameId="includeInGames"
-              options={[
-                { label: 'Yes', value: 'true' },
-                { label: 'No', value: 'false' },
-              ]}
-            />
-          </div>
+          <div
+            className={`mt-6 grid grid-cols-12 gap-6 ${
+              mediaType === AUDIO ? 'col-span-12' : 'col-span-6'
+            }`}
+          >
+            <div className="col-span-12 sm:col-span-6">
+              <Form.TextField
+                label="Title"
+                nameId="title"
+                register={register}
+                errors={errors}
+              />
+            </div>
+            <div className="col-span-12">
+              <Form.TextField
+                label="Description"
+                nameId="description"
+                register={register}
+                errors={errors}
+              />
+            </div>
 
-          <div className="col-span-12">
-            <Form.RadioButtons
-              label="Include on the Kids site?"
-              control={control}
-              errors={errors}
-              nameId="includeInKids"
-              options={[
-                { label: 'Yes', value: 'true' },
-                { label: 'No', value: 'false' },
-              ]}
-            />
-          </div>
+            {mediaType === AUDIO && (
+              <div className="col-span-12">
+                <Form.AutocompleteMultiple
+                  label="Speakers"
+                  nameId="speakerIds"
+                  control={control}
+                  options={speakerOptions}
+                  placeholder="Find speakers to add.."
+                />
+              </div>
+            )}
+
+            <div className="col-span-12">
+              <Form.TextField
+                label="Acknowledgements"
+                nameId="acknowledgement"
+                register={register}
+                errors={errors}
+              />
+            </div>
+
+            <div className="col-span-12">
```

Comment:
One way we could reduce duplication here, and in other forms, would be to create a Form component for Audience that contains the kids and games toggles.